### ISSUE TITLE
[Testing] Comparison of Accuracy Performance (mAP) between Default Parameters and Scaled Parameters

### ISSUE THREAD

**Issue by jaskiratsingh2000** [jaskiratsingh2000/Research-Scalable-Vehicle-Detection-on-Edge-Devices#15]
The [yolov5s.yaml](https://github.com/ultralytics/yolov5/blob/master/models/yolov5s.yaml) configuration file inside the ultralytics/yolov5 repo contains various scaling parameters. Hence I scaled the **depth_multiple** and **width_multiple** values while experimented the performance evaluation.

**Default Values:**

```
depth_multiple: 0.33
width_multiople: 0.50
```

**Scaled Values:**

```
depth_multiple: 0.01
width_multiople: 0.02
```

**Comment by jaskiratsingh2000** [comment]
I trained the [Mask Wearing Dataset](https://public.roboflow.com/object-detection/mask-wearing) with the pre-trained `--weights 'yolov5s.pt' --batch-size  10 --epochs 50`

The time took to **train the dataset with default parameters** is **4.256 hours** and to **train the dataset with scaled parameters** is **0.396 hours**

**Comment by jaskiratsingh2000** [comment]
**Performance Accuracy results with Default Parameters:**


```
Fusing layers... 
Model Summary: 224 layers, 7266973 parameters, 0 gradients
val: Scanning 'valid/labels.cache' images and labels... 29 found, 0 missing
val: Scanning 'valid/labels.cache' images and labels... 29 found, 0 missing
               Class     Images     Labels          P          R     mAP@.5
                 all         29          0          0          0          0          0
Speed: 15.9ms pre-process, 3107.6ms inference, 6.8ms NMS per image at shape (32, 3, 640, 640)
Results saved to runs/test/exp
```


**Performance Accuracy results with Scaled Parameters:**


```
Fusing layers... 
Model Summary: 224 layers, 7266973 parameters, 0 gradients
val: Scanning 'valid/labels.cache' images and labels... 29 found, 0 missing
val: Scanning 'valid/labels.cache' images and labels... 29 found, 0 missing
val: Scanning 'valid/labels.cache' images and labels... 29 found, 0 missing
               Class     Images     Labels          P          R     mAP@.5
                 all         29          0          0          0          0          0
Speed: 15.3ms pre-process, 2286.9ms inference, 4.7ms NMS per image at shape (32, 3, 640, 640)
Results saved to runs/test/exp
```

**Comment by jaskiratsingh2000** [comment]
Didn't come out to be fruitful as mAP value is 0
